vars/main.yml

```diff
@@ -54,3 +54,5 @@ php_cli_ondrej_mods_dependencies:
     - php-ast
   sqlite3:
     - "php{{ php_cli_ondrej_version }}-sqlite3"
+  libsodium:
+    - "php{{ php_cli_ondrej_version }}-libsodium"
```

Comment:
I don't think this package has a version number, just like `php-ast` for instance. Can you fix it?